### PR TITLE
Include Portable PDBs in nuget packages

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

**Comment by tmat** [comment]
@jasonmalinowski @jaredpar

File: src/NuGet/Microsoft.CodeAnalysis.Build.Tasks.nuspec [hunk 1]
```diff
@@ -51,7 +51,9 @@
     </contentFiles>
   </metadata>
   <files>
+    <!-- Include PDB unless embedded into DLL -->
     <file src="Dlls\MSBuildTask\Microsoft.Build.Tasks.CodeAnalysis.dll" target="lib\netstandard1.3" />
+    <file src="Dlls\MSBuildTask\Microsoft.Build.Tasks.CodeAnalysis.pdb*" target="lib\netstandard1.3" />
```
Comment (jaredpar) [inline src/NuGet/Microsoft.CodeAnalysis.Build.Tasks.nuspec]: why do we need the `*` here?

Comment (tmat) [reply src/NuGet/Microsoft.CodeAnalysis.Build.Tasks.nuspec]: To make build work locally and on Jenkins, where the PDBs are embedded.

Comment (jaredpar) [reply src/NuGet/Microsoft.CodeAnalysis.Build.Tasks.nuspec]: Gotcha. Using a glob to allow it to be excluded without error.

Comment (tmat) [reply src/NuGet/Microsoft.CodeAnalysis.Build.Tasks.nuspec]: Yup.

**Comment by tmat** [comment]
Test failure: https://github.com/dotnet/roslyn/issues/23042

**Comment by tmat** [comment]
test ubuntu_16_debug_prtest please

**Comment by tmat** [comment]
FYI @ctaggart

File: src/NuGet/Microsoft.CodeAnalysis.Build.Tasks.nuspec (hunk 1 above)
Comment (jasonmalinowski) [reply src/NuGet/Microsoft.CodeAnalysis.Build.Tasks.nuspec]: @tmat @jaredpar Can we add that explanation in a comment please? :smile:

Comment (tmat) [reply src/NuGet/Microsoft.CodeAnalysis.Build.Tasks.nuspec]: There is a comment.

**Comment by tmat** [comment]
NuGet doesn't do anything special with them right now. They will end up in the nuget cache next to the dll, where the debugger finds them if the dll is loaded form nuget cache. When developing .NET Core app the CLR loads the dlls from nuget cache, so it will just work. If it doesn't just work (in other scenarios, where the dlls are loaded from elsewhere) you can manually point the debugger to the PDB in the package. we need x-plat symbol server support to improve such scenarios.

**Comment by tmat** [comment]
@vancem @noahfalk FYI

**Pull request merged**